operators/_regex.yaml

```diff
@@ -0,0 +1,75 @@
+id: _regex
+type: PageSiderMenu
+public: true
+properties:
+  title: Nunjucks
+
+actions:
+  onInit:
+    - id: set_state
+      type: setState()
+      params:
+        string: A string value
+        boolean: true
+        # date:
+        #   _date: now
+        array_primitive:
+          - 1
+          - 2.5
+          - 3000
+        array_object:
+          - test: true
+            name: Adam
+            value: 10
+          - test: false
+            name: Bob
+            value: 50
+          - test: true
+            name: Carl
+            value: 1000
+        object:
+          string: text
+          boolean: false
+          date:
+            _date: "2000-01-01"
+          array:
+            - A
+            - B
+            - C
+
+blocks:
+  _ref: 
+    path: operators/template_operators.yaml
+    vars:
+      operator_name: _regex
+      operator_title: Regex
+      description_content: |
+        The `_regex` operator executes a regular expression to match a sequence of characters that define a search pattern and returns true is the pattern is matched or false if it is not.. [Read more](https://medium.com/factory-mind/regex-tutorial-a-simple-cheatsheet-by-examples-649dc1c3f285) or [play around](https://regex101.com/).
+
+        #### Parameters
+        `_regex` takes three parameters but can also be written in short-hand notation. The parameters are:
```

Comment:
`_regex` takes three parameters or can be written in short-hand notation when refering to state.